CONTRIBUTING.md

```diff
@@ -86,6 +86,9 @@ Official high quality brand logos and brand colors can usually be found in the f
 1. Wikimedia (which should provide a source)
 1. GitHub repositories
 
+It may be the case that no official source exist, but an unofficial icon has gained widespread acceptance and popularity. In such cases the unofficial icon can be included, but the details will be judged on a case-by-base basis. The JavaScript icon is an example of this.
```

Comment:
```suggestion
It may be the case that no official source exists, but an unofficial icon has gained widespread acceptance and popularity. In such cases the unofficial icon can be included, but the details will be judged on a case-by-case basis. The JavaScript icon is an example of this.
```

Just two grammar fixes

Reply:
good catch! Committed the changes already.